rbac/management/principal/view.py

```diff
@@ -188,7 +187,7 @@ def get(self, request):
                 "first": f"{path}?limit={limit}&offset=0{usernames_filter}",
                 "next": f"{path}?limit={limit}&offset={offset + limit}{usernames_filter}",
                 "previous": f"{path}?limit={limit}&offset={previous_offset}{usernames_filter}",
-                "last": None,
+                "last": f"{path}?limit={limit}&offset={int(limit - int(count))}{usernames_filter}",
```

Comment:
if limit < count then offset will be negative

maybe we can reuse the `get_paginated_response()` from [rbac/api/common/pagination.py](https://github.com/RedHatInsights/insights-rbac/blob/af853082447418d58a1cdde5e8bbdf350f8ba0ae/rbac/api/common/pagination.py#L80)

Reply:
@EvanCasey13 is it possible to leverage `get_paginated_response ` somehow ?
it looks there some cases which needs to covered.